src/neuroconv/tools/nwb_helpers.py

```diff
@@ -166,12 +167,15 @@ def make_or_load_nwbfile(
         elif nwbfile is None:
             nwbfile = make_nwbfile_from_metadata(metadata=metadata)
         yield nwbfile
+    except Exception as e:
+        success = False
+        raise e
     finally:
         if nwbfile_path:
             try:
                 io.write(nwbfile)
 
-                if verbose:
+                if success and verbose:
```

Comment:
Should the file still be written if (line 176) if an exception was risen?

Reply:
Probably not 😆

Reply:
Line 172 now in this preview somehow.